README.md

```diff
@@ -0,0 +1,62 @@
+# Mapserver OSM basemaps
```

Comment:
You should make a PR in the mapserver/docs repository! :)

Reply:
ok!

Reply:
At least a PR in Docs yes. The readme could link to the doc yes!